python/tests/test_extend.py

```diff
@@ -18,10 +18,6 @@ def test_extend(self):
             [[],
              [ValueAtom(5)],
              [ValueAtom('B')]])
-        self.assertEqual(
-              metta.run('! &runner')[0][0].get_object().value,
```

Comment:
Did you remove it by accident or it doesn't work after changes.

Reply:
I removed it intentionally because the runner object is actually a handle to the runner.  Two runners are different boxed objects but they point to the same runner.

Reply:
Do you think checking MeTTa pointer equality is important enough that it makes sense to add a "metta_eq" function to C?  My thinking is that it probably doesn't matter except for this test.

Reply:
Ultimately I figured comparing the referenced runner is worthwhile functionality.  Added in 8254d3f